src/Sylius/Bundle/WebBundle/Resources/translations/menu.fr.yml

```diff
@@ -87,6 +87,8 @@ sylius:
         login: Connexion
         logout: Déconnexion
         register: Créer un compte
+      sidebar:
```

Comment:
This should be reverted. New translation are added only to `en`, then it's updated on [Crowdin](http://translate.sylius.org/) & if translated there, it's auto updated in repository (from time to time)...